playground/frontend/pubspec.yaml

```diff
@@ -46,11 +46,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
-
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
-  cupertino_icons: ^1.0.2
+  provider: ^6.0.0
```

Comment:
@pabloem I checked that the [provider v.6.0.1 package license](https://pub.dev/packages/provider/license) is MIT.